iothub/device/samples/PnpDeviceSamples/PnpDeviceSamples/SimpleThermostat/Models/SimpleThermostat.json

```diff
@@ -0,0 +1,53 @@
+﻿{
+  "@context": "dtmi:dtdl:context;2",
```

Comment:
This interface is taken from here: https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/SimpleThermostat.json